pwndbg/wrappers/readelf.py

```diff
@@ -33,3 +34,49 @@ def _extract_jumps(line):
             return False
     except IndexError:
         return False
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def get_load_segment_info():
+    '''
+    Looks for LOAD sections by parsing the output of `readelf -l <binary>`
+    '''
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [get_jmpslots.cmd_path, "-l", local_path]
```

Comment:
Please change `'-l'` to `--program-headers` both here and in the docstring so it would be easier to understand what is this flag for :)